WcaOnRails/config/locales/en.yml

```diff
@@ -269,10 +269,10 @@ en:
         refund_policy_percent: "Percent to be refunded"
         refund_policy_limit_date: "Limit date for refunds"
         guests_entry_fee_lowest_denomination: "Entry fee for guests"
-        regulation_z1: "I would like to require competitors to submit puzzles during registration as per Regulation Z1)"
-        regulation_z1_reason: "Reason for Z1)"
-        regulation_z3: "I would like to select competitors who directly qualify for certain rounds of certain events based on the results of specific previous competitions as per Regulation Z3)"
-        regulation_z3_reason: "Reason for Z3)"
+        regulation_z1: "I would like to require competitors to submit puzzles at a specific time before an event is scheduled to begin"
```

Comment:
Shouldn't we rename these fields on the database know that Regulations Zn) don't exist anymore?

Reply:
Probably,
However, regulation_z1_reason and regulation_z3_reason have 48 uses each, regulation_z1 and regulation_z3 have 97 uses each.
This is a bit much for me to refactor at this time without really knowing what I'm doing.

Reply:
Many of their uses just come from the translations, so you don't need to bother with those, you just need to change the uses on the actual code and on the English locale file.

Reply:
This would also involve changing it in the database correct?

Reply:
Correct.

Reply:
There's already a lot of different stuff going on in this PR. I'd vote for renaming these in a separate PR.

Reply:
Yep, they will be renamed on a different PR. We already have an issue to track this: #4217.